server/src/main/java/com/sauron/model/entities/User.java

```diff
@@ -24,17 +27,21 @@
     @Column(unique = true, nullable = false)
     private String email;
 
-    @OneToMany(mappedBy = "user")
-    private Set<BankAccount> bankAccounts = new HashSet<>();
+    @ElementCollection
+    @CollectionTable(
+            name = "BANK_CONNECTION_DATA",
+            joinColumns = @JoinColumn(name = "USER_ID")
+    )
+    private Set<BankConnectionData> bankConnectionData = new HashSet<>();
```

Comment:
The name of a field seems a little bit awkward to me. The class name is BankConnectionData and set of those objects is also bankConnectionData :P I know there is no plurar form for data, but still... Another argument for finding different name for this class :)